tests/model_hub_tests/models_hub_common/test_performance_model.py

```diff
@@ -0,0 +1,220 @@
+            results.status = Status.GET_READ_MODEL
+            read_model = self.compile_model(self.get_read_model(model_obj), ie_device)
+            print("Infer the converted model")
+            results.status = Status.INFER_CONVERTED_MODEL
+            converted_model_time, converted_model_time_variance = self.infer_model(converted_model, inputs)
+            print('converted model time infer {}'.format(converted_model_time))
+            print('converted model time infer var {}'.format(converted_model_time_variance))
+            print("Infer read model")
+            results.status = Status.INFER_READ_MODEL
+            read_model_time, read_model_time_variance = self.infer_model(read_model, inputs)
+            print('read model time infer {}'.format(read_model_time))
+            print('read model time infer var {}'.format(read_model_time_variance))
+
+            infer_time_ratio = converted_model_time/read_model_time
+
+            results.converted_infer_time = converted_model_time
+            results.converted_model_time_variance = converted_model_time_variance
+            results.read_model_infer_time = read_model_time
+            results.read_model_infer_time_variance = read_model_time_variance
+            results.infer_time_ratio = infer_time_ratio
+
+            if abs(infer_time_ratio - 1) > TestModelPerformance.threshold_ratio:
+                if (read_model_time_variance > TestModelPerformance.threshold_var
+                        or converted_model_time_variance > TestModelPerformance.threshold_var):
+                    results.status = Status.LARGE_INFER_TIME_DIFF_WITH_LARGE_VAR
+                    results.error_message = "too large ratio {} with large variance".format(infer_time_ratio)
+                else:
+                    results.status = Status.LARGE_INFER_TIME_DIFF
+                    results.error_message = "too large ratio {}".format(infer_time_ratio)
+            else:
+                results.status = Status.OK
+        except:
+            ex_type, ex_value, tb = sys.exc_info()
+            results.error_message = "{tb}\n{ex_type}: {ex_value}".format(tb=''.join(traceback.format_tb(tb)),
+                                                             ex_type=ex_type.__name__, ex_value=ex_value)
+        return results
+
+    def run(self, model_name, model_link, ie_device):
+        self.result = Results()
+        t0 = time.time()
+        self.result = multiprocessing_run(self._run, [model_name, model_link, ie_device], model_name, self.infer_timeout)
+        t1 = time.time()
+        print('test running time {}'.format(t1 - t0))
+        if self.result.status == Status.OK:
+            return
+        err_message = "\n{func} running failed: \n{msg}".format(func=model_name, msg=self.result.error_message)
+        if self.result.status == Status.LARGE_INFER_TIME_DIFF_WITH_LARGE_VAR:
+            pytest.xfail(err_message)
+        else:
+            pytest.fail(err_message)
```

Comment:
Not sure that this is correct way because we always mark failing tests as xfailed (expected failure). New failures will not be expected for us.

Reply:
should we use skip or something like that?
these failures have to be ignored due to unstable inference time

Reply:
About such cases we should know of in advance. We should mark them in `precommit` file as:
```
model_name,model_link,skip,Reason
```

Reply:
Unfortunately, we can' detect it in advance, this measurement can be done only during runtime.
If we do not skip such tests with this criteria , it can affect pre-commit stability